src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -427,7 +428,9 @@ export function Autocomplete<OptionType = IDefaultOptionType>(props: IProps<Opti
     onChange: handleOnChange,
     formatGroupLabel: formatGroupLabel,
     selectWidth: width,
-    options: items
+    options: items,
+    closeMenuOnSelect: !isMulti,
+    isMulti: isMulti
```

Comment:
This can just be `isMulti`. No need for `key: value` designation

Reply:
Ah right right.